api/all/src/test/java/io/opentelemetry/api/internal/ConfigUtilTest.java

```diff
@@ -56,4 +64,45 @@ void defaultIfnull() {
     assertThat(ConfigUtil.defaultIfNull("val1", "val2")).isEqualTo("val1");
     assertThat(ConfigUtil.defaultIfNull(null, "val2")).isEqualTo("val2");
   }
+
+  @Test
+  @SuppressWarnings("ReturnValueIgnored")
+  void systemPropertiesConcurrentAccess() throws ExecutionException, InterruptedException {
```

Comment:
This test fails on java 8 if you call `System.getProperties()` instead of `ConfigUtil#safeSystemProperties()`.